schema/encoding/jsonschema/jsonschema.go

```diff
@@ -101,12 +101,20 @@ func validatorToJSONSchema(w io.Writer, v schema.FieldValidator) (err error) {
 		if t.ValuesValidator != nil {
 			ew.writeString(`, "items": `)
 			if ew.err == nil {
-				ew.err = validatorToJSONSchema(w, t.ValuesValidator)
+				ew.err = validatorToJSONSchema(w, t.ValuesValidator, true)
```

Comment:
don't know if it covers all the use cases, if you can work a patch that makes it work though that'd be great.